static/src/stylesheets/layout/new-header/_veggie-burger.scss

```diff
@@ -29,26 +29,34 @@
         right: $gutter-xlarge;
     }
 
+    @include mq(desktop) {
+        &:focus {
```

Comment:
So visually the focus should be on this element, but actually for tab users the focus shouldn't be. See my closed PR for how you should do this: https://github.com/guardian/frontend/pull/17177/files